cpp/models/abm/mask_type.h

```diff
@@ -33,7 +33,8 @@ namespace abm
  */
 enum class MaskType : std::uint32_t
 {
-    Community = 0,
+    None = 0,
```

Comment:
It was used to replace the previous Community level (`=0`). I think it's more of a convention since all the other types (LocationType, InterventionType). Since `uint32_t` starts with 0 so I guess I could remove all the `=0`.

Reply:
Per standard, the first enumerator will always be set to zero. So the `=0` won't cause any harm. But it makes it look like the order or specific values may be important somewhere in the code, which AFAIK is not the case (i.e. we should be able to change the order of mask types without changing the computed results).

If we want to explicitly fix the order of the enum, the =0 can stay, but it would be preferable to also have an explanation as to why the order is important.

Reply:
Ok. I now removed all unnecessary `=0` from all the files where the order or specific values are not important. Should be cleaner this way.